Gems/Atom/Tools/MaterialCanvas/Assets/MaterialCanvas/TestData/mul.materialcanvasnode.azasset

```diff
@@ -0,0 +1,81 @@
+{
+    "Type": "JsonSerialization",
+    "Version": 1,
+    "ClassName": "DynamicNodeConfig",
+    "ClassData": {
+        "category": "Math Functions",
+        "title": "mul",
+        "inputSlots": [
+            {
+                "name": "inValue1",
+                "displayName": "Value1",
+                "description": "Value1",
+                "supportedDataTypes": [
+                    "float",
```

Comment:
Matrix types maybe aren't supported yet?

Reply:
Just not exposed. Need to discuss how we want to expose them in the node and UI.